v4/source/channels.yaml

```diff
@@ -851,6 +849,62 @@
             // SearchChannels
             channels, resp := Client.SearchChannels(<TEAMID>, search)
 
+  '/teams/{team_id}/channels/search_archived':
+    post:
+      tags:
+        - channels
+      summary: Search archived channels
+      description: |
+        Search archived channels on a team based on the search term provided in the request body.
+        ##### Permissions
+        Must have the `list_team_channels` permission.
+
+        In server version 5.16 and later, a user without the `list_team_channels` permission will be able to use this endpoint, with the search results limited to the channels that the user is a member of.
```

Comment:
This description should include `__Minimum server version__: 5.18` (assuming this makes it in before feature cutoff) since it's a new endpoint